dirty_cat/_feature_augmenter.py

```diff
@@ -70,9 +70,11 @@ def __init__(
         self,
         tables: dict,
         main_key: str,
+        match_score=0,
```

Comment:
```suggestion
        match_score: float = 0.0,
```
Also, as it's a new parameter in the class, it needs to be added to the docstring.